cli/azd/pkg/apphost/generate.go

```diff
@@ -431,15 +468,48 @@ func (b *infraGenerator) Compile() error {
 		b.bicepContext.ContainerApps[name] = cs
 	}
 
-	for projectName, project := range b.projects {
+	for resourceName, docker := range b.dockerfiles {
+		projectTemplateCtx := genContainerAppManifestTemplateContext{
+			Name: resourceName,
+			Env:  make(map[string]string),
+		}
+
+		binding, err := validateAndMergeBindings(docker.Bindings)
+		if err != nil {
+			return fmt.Errorf("configuring ingress for resource %s: %w", resourceName, err)
+		}
+
+		if binding != nil {
```

Comment:
Followed you lead here, @weikanglim.  When you land your changes, I will rebase and see if I can merge these two cases and simplify.